net/genders/Portfile

```diff
@@ -16,11 +15,10 @@ long_description    Genders is a static cluster configuration database \
                     used for cluster configuration management. It is used by a \
                     variety of tools and scripts for management of large clusters.
 
-checksums           rmd160  819ea7a2e8ad552815c5446037554a6871ca9ae8 \
-                    sha256  0ff292825a29201106239c4d47d9ce4c6bda3f51c78c0463eb2634ecc337b774 \
-                    size    595795
-
-master_sites        ${homepage}/releases/download/${github.project}-${github.tag_prefix}
+checksums           rmd160  c39e9d113ab01d34ce81ddfde72b989411fa441e \
+                    sha256  0dc186ec8fd01ec10b5e171d8b8ef632a56d17c135baee2570c0a3c0ffd9d64d \
+                    size    1239423
+master_sites        ${homepage}/archive/refs/tags/${github.project}-${github.tag_prefix}
```

Comment:
this should really be set by the `github` PortGroup already, why do you need to override this?